level1/f256/modules/vtio.asm

```diff
@@ -1163,6 +1216,10 @@ GetStat             cmpa      #SS.EOF             is this the EOF call?
                     lbeq      GSKySns             branch if so
                     cmpa      #SS.Joy             get joystick position?
                     beq       SSJoy               branch if so
+		    ifgt      Level-1
```

Comment:
Why is this conditional here if we support Level 1?

Reply:
Removed support for Level-1 to remove from FEU. Since FEU uses L1 vtio, it was easier to strip the mouse from L1 than to make a conditional for the FEU.